go.mod

```diff
@@ -87,7 +87,7 @@ require (
 	github.com/jackc/pgx/v4 v4.18.1
 	github.com/jcmturner/gokrb5/v8 v8.4.4
 	github.com/johannesboyne/gofakes3 v0.0.0-20210217223559-02ffa763be97
-	github.com/jonboulle/clockwork v0.3.0
+	github.com/jonboulle/clockwork v0.4.0
```

Comment:
This might cause problems: https://github.com/jonboulle/clockwork/pull/55/files.

Aren't breaking changes fun?